### PR TITLE
ENH: Upgrade SDCFlows to new API (1.0.0rc1)

### LINKED ISSUE #1884: PIN: SDCFlows 1.0.0rc1 / compatibility with refactored API

### PR DESCRIPTION
**Pull request by oesteban**
Closes #1884.

### CODE REVIEW

**Comment by oesteban** [comment]
Please note the new condensed view for phase1/2-phasediff-fieldmap kind of fieldmaps (ds054). This PR merges together the magnitude segmentation view, the magnitude registration view and the fieldmap registration view (hopefully you like the new one better).

I think we still need to add a colorbar in Hz to the fieldmap overlay, and make it symmetric bc right now it is a bit confusing. But overall I believe it is much better.

**Comment by oesteban** [comment]
Another focus point (esp. for @effigies) is poldracklab/sdcflows#63, which is necessary for this PR to work.

**Comment by oesteban** [comment]
Last comment - this PR should be then followed by addressing #1892, ensuring we add MNI152NLin2009cAsym to the standard spaces list if SyN is to run.

**Comment by oesteban** [comment]
Should also fix #1665.

**Comment by AKSoo** [comment]
The actual fieldmaps are fine, but the fieldmap report figures have these values spilling over the y-axis.

yspill

**Comment by oesteban** [comment]
That extrapolation is added by FUGUE. I agree we should open an issue in SDCFlows about it, to double-check that the extrapolation is done once the fieldmap has been mapped into the target EPI.

The alternative to this, once again, is implementing poldracklab/sdcflows#14 so that extrapolation is less of a blanket across the PE-axis solution.

**Comment by effigies** [comment]
Sorry, missed this in a flood of notifications... I'll review now.

File: setup.cfg [hunk 1]
```diff
@@ -79,6 +79,7 @@ fmriprep =
     data/boilerplate.bib
     data/itkIdentityTransform.txt
     data/flirtsch/bbr.sch
+    data/reports-spec.yml
```
Comment (effigies) [inline setup.cfg]: For this to correctly work, we might need to remove `include_package_data = True`. I've recently discovered (https://github.com/niflows/nipype1-workflows/pull/3, https://github.com/nipy/nibabel/commit/7120347111d0f0a2af29bb9ba2d133f61fdb17b8) that that line seems to mean *only* use `MANIFEST.in`.

Comment (oesteban) [reply setup.cfg]: If I understand correctly, we either set `include_package_data = False` or add all these objects in the `MANIFEST.in`?

**Comment by oesteban** [comment]
Okay, updated the pin to a released version since the patch is already in SDCFlows/master. That was breaking builds.

File: setup.cfg (hunk 1 above)
Comment (effigies) [reply setup.cfg]: I think so.

OTOH I think these must have worked previously...

Comment (oesteban) [reply setup.cfg]: Moving the conversation to #1896

**Pull request merged**